deploy/database/data.button.sql

```diff
@@ -1152,3 +1152,315 @@ UPDATE button
+UPDATE button SET flavor_text="Gilly is a perky goth who likes to play games. She enjoys just about everything, just about everything else, and beating people up. (dressed as Clare)" WHERE name="Gilly";
+UPDATE button SET flavor_text="Igor is a vicious, sweaty man with a garbage can on his head. He likes long walks in the park, red rhubarb jelly and beating people up. (dressed as Hammer)" WHERE name="Igor";
+UPDATE button SET flavor_text="Ken is a hard-riding hobbyist with a heart of gold. He likes frivolity, jumping up and down, running with his tongue hanging out, and beating people up. (dressed as Bauer)" WHERE name="Ken";
+UPDATE button SET flavor_text="Matt is as loveable as the day is long. Just look at that cute widdow nose. He likes role-playing, the unexpected, and beating people up. (dressed as Hannah)" WHERE name="Matt";
+UPDATE button SET flavor_text="Sven is a Trow (troll) who likes to trade recipes with the other queens at the leather bar, swap housekeeping tips with his neighbors, prune roses, and beat people up." WHERE name="Sven";
+UPDATE button SET flavor_text="Yseulte is a Fuah (bad water faery) who lives by the Maine Avenue fish market. She likes to party with the Grrls, ride vintage Harleys, smear women's makeup, collect scallop shells, and beat people up." WHERE name="Yseulte";
+UPDATE button SET flavor_text="Aldric can belch the anthem of his home town and once drank an entire tankard of beer. He likes damsels, horses, and beating people up." WHERE name="Aldric";
+UPDATE button SET flavor_text="Cesare is the banished heir to the throne of a large kingdom. He likes plotting revenge, procrastinating people, and beating people up." WHERE name="Cesare";
+UPDATE button SET flavor_text="Chang is far less intelligent than most people think he is. He likes long naps, pieces of string, and beating people up." WHERE name="Chang";
+UPDATE button SET flavor_text="Elihu senses danger at all times, which makes him tense and irritable. His mind is filled with trivial detail, and he loves to beat people up." WHERE name="Elihu";
+UPDATE button SET flavor_text="Farrell spends his weekends by the seaside, watching re-runs and knitting sweaters. But during the week, he likes to beat people up." WHERE name="Farrell";
+UPDATE button SET flavor_text="Frasquito is a very short person and the self-proclaimed inventor of lye. He likes fruit trees and oysters, and he likes to beat people up." WHERE name="Frasquito";
+UPDATE button SET flavor_text="Lark is a triple Mud Baron on the Planet of the Bird People. He likes waffling, collecting shiny objects, and beating people up." WHERE name="Lark";
+UPDATE button SET flavor_text="Luce is a healer and a clever prestidigitator. She can keep children amused for hours, using only a pencil, and she likes to beat people up." WHERE name="Luce";
```

Comment:
Whoa, Comma Ninja actually thinks you should remove a comma.  This is not common.  s/She can keep children amused for hours, using only a pencil/She can keep children amused for hours using only a pencil/